packages/atomic/src/components/atomic-search-box/atomic-search-box.tsx

```diff
@@ -552,7 +596,11 @@ export class AtomicSearchBox {
             el?.replaceChildren(item.content);
           }
         }}
-        {...(isSuggestionElement(item) && {'data-query': item.query})}
+        {...(isSuggestionElement(item) && {
+          role: 'option',
```

Comment:
I would only add the roles `listbox` and `option` to the left container since it changes the query and corresponds the most with what we think of as suggestions.

Reply:
if you plug in the existing components that we have, this is what will happen (actually, I think I did it wrong, it should be 
```
isSuggestionElement(item) && item.query && {
          role: 'option',
```
)
but I am thinking of times when people might attach their own suggestions to one panel or another. do you agree that we should support the `option="role"` in the right panel to account for those cases?